node/hack/governor/src/index.ts

```diff
@@ -162,6 +162,11 @@ axios
                   continue;
                 }
               }
+
+              // If the character list is violated, then skip the coin. The error is logged in the function if something happens to have some sort of check on it.
+              if(!(await safetyCheck(chain, wormholeAddr, data.Symbol, data.CoinGeckoId, data.TokenDecimals, data.TokenPrice, data.Address, notional))){
```

Comment:
Besides logging to the console can we also add the tokens that violate the safety check to the txt summary output? Since this runs in CI most of the time nobody is going to read the console output unless we flag that tokens have failed the safety check

Reply:
Added this as well. Good catch!